Add new Windows machines for pull requests and build pipelines

Four new Windows 2012R2 machines have been created and in need configuring with the ansible scripts from AdoptOpenJDK.

1. oj9-win2012r2-1; 169.55.170.69
1. oj9-win2012r2-2; 169.55.150.148
1. oj9-win2012r2-3; 169.55.170.74
1. oj9-win2012r2-4; 169.55.150.158

Administrator login passwords available under private communication.

Each one is 8 CPU, 8GB RAM, Windows 2012R2, 100Gb boot disk, 25 Gb data disk, 100 Mbs network link.

- [x] PR for JenkinsFiles #1778 
- [x] Ansible machines (1/4 done)
- [x] Connect to Jenkins (1/4 done)
- [x] Setup Build jobs
- [x] Setup test jobs
- [x] Setup Pipeline jobs
- [x] Setup PR jobs
- [x] README

Got compiles working
Had to change a few things on the machine
- Install openssh through Cygwin installer ([reference](https://go.cloudbees.com/docs/cloudbees-documentation/admin-cje/agents/#_microsoft_windows))
- Uninstall Git for Windows
- Install Git through Cygwin installer
- Add Shortname for "Program Files (x86)" directory ([reference](https://superuser.com/questions/681330/how-to-force-short-name-8dot3-generation))
   - fsutil 8dot3name query      # returned 2
   - fsutil behavior set disable8dot3 0
   - fsutil file setshortname "Program Files (x86)" PROGRA~2
   - dir /X      # confirm shortname created
   - fsutil behavior set disable8dot3 2
- Install x86_64-w64-mingw32-g++ through Cygwin installer

Current issue is trying to compile the test material ([link](https://ci.eclipse.org/openj9/view/Test/job/Test-Extended-JDK10-win_x86-64_cmprssptrs/2/console))
```
00:04:25.441 clean:
00:04:25.441    [delete] Deleting directory C:\cygwin64\home\jenkins\workspace\Test-Extended-JDK10-win_x86-64_cmprssptrs\openj9\test\functional\VM_Test\bin
00:04:25.441 
00:04:25.441 BUILD FAILED
00:04:25.441 C:\cygwin64\home\jenkins\workspace\Test-Extended-JDK10-win_x86-64_cmprssptrs\openj9\test\TestConfig\scripts\build_test.xml:71: The following error occurred while executing this line:
00:04:25.441 C:\cygwin64\home\jenkins\workspace\Test-Extended-JDK10-win_x86-64_cmprssptrs\openj9\test\functional\build.xml:62: The following error occurred while executing this line:
00:04:25.441 C:\cygwin64\home\jenkins\workspace\Test-Extended-JDK10-win_x86-64_cmprssptrs\openj9\test\functional\VM_Test\build.xml:139: The following error occurred while executing this line:
00:04:25.441 C:\cygwin64\home\jenkins\workspace\Test-Extended-JDK10-win_x86-64_cmprssptrs\openj9\test\functional\VM_Test\build.xml:134: java.nio.file.InvalidPathException: Illegal char <*> at index 113: C:\cygwin64\home\jenkins\workspace\Test-Extended-JDK10-win_x86-64_cmprssptrs\openj9\test\functional\VM_Test\data\*.class
```

Next issue:
```
[C:\cygwin64\home\jenkins\workspace\PullRequest-Compile-JDK8-win_x86-64_cmprssptrs-OpenJDK8] Running shell script
+ make all
Building OpenJDK for target 'all' in configuration 'windows-x86_64-normal-server-release'

## Starting langtools
Compiling 2 files for BUILD_TOOLS
Could not open temporary file for writing! c:/cygwin/tmp\atfile_a10924
make[1]: *** No rule to make target 'all', needed by 'default'.  Stop.
make: *** [/home/jenkins/workspace/PullRequest-Compile-JDK8-win_x86-64_cmprssptrs-OpenJDK8//make/Main.gmk:87: langtools-only] Error 2
```
Not sure why its trying to write to c/cygwin/ without the `64`

--- a/buildenv/jenkins/jobs/pipelines/Pipeline-OMR-Acceptance
+++ b/buildenv/jenkins/jobs/pipelines/Pipeline-OMR-Acceptance
@@ -21,7 +21,7 @@
  *******************************************************************************/
 
 def SDK_VERSIONS = ['8', '10']
-def SPECS = ['linux_ppc-64_cmprssptrs_le', 'linux_390-64_cmprssptrs', 'aix_ppc-64_cmprssptrs', 'linux_x86-64_cmprssptrs', 'linux_x86-64']
+def SPECS = ['linux_ppc-64_cmprssptrs_le', 'linux_390-64_cmprssptrs', 'aix_ppc-64_cmprssptrs', 'linux_x86-64_cmprssptrs', 'linux_x86-64', 'win_x86-64_cmprssptrs']
 
 def OPENJDK_REPOS = [:]
 OPENJDK_REPOS['8'] = [repo: 'https://github.com/ibmruntimes/openj9-openjdk-jdk8.git', branch: 'openj9']

